pippo-security-parent/pippo-pac4j/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>ro.pippo</groupId>
+        <artifactId>pippo-security-parent</artifactId>
+        <version>1.7.0-SNAPSHOT</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+    <artifactId>pippo-pac4j</artifactId>
+    <name>Pippo Pac4j</name>
+    <description>Pac4j integration</description>
+
+    <properties>
+        <pac4j.version>2.1.0</pac4j.version>
```

Comment:
I released pac4j v2.2.0 a few days ago, it's fully backward compatible and integrate several SAML fixes/improvements.

Reply:
Updated to v2.2.0